mlflow/langchain/runnables.py

```diff
@@ -200,14 +201,36 @@ def _load_runnable_branch(file_path: Union[Path, str], model_type: str):
     return RunnableBranch(*branches)
 
 
+def _load_runnable_assign(file_path: Union[Path, str]):
+    """Load the model
+
+    Args:
+        file_path: Path to file to load the model from.
+    """
+    from langchain.schema.runnable.passthrough import RunnableAssign
+
+    # Convert file to Path object.
+    load_path = Path(file_path) if isinstance(file_path, str) else file_path
+    if not load_path.exists() or not load_path.is_dir():
+        raise MlflowException(
+            f"File {load_path} must exist and must be a directory in order to load runnable."
+        )
+
+    mapper_file = load_path / _MAPPER_FOLDER_NAME
+    if not mapper_file.exists() or not mapper_file.is_dir():
+        raise MlflowException(
+            f"Folder {mapper_file} must exist and must be a directory "
+            "in order to load runnable assign with mapper."
+        )
+    mapper = _load_runnable_with_steps(mapper_file, "RunnableParallel")
+    return RunnableAssign(mapper)
+
+
 def _save_internal_runnables(runnable, path, loader_fn, persist_dir):
     conf = {}
     if isinstance(runnable, lc_runnables_types()):
         conf[_MODEL_TYPE_KEY] = runnable.__class__.__name__
         conf.update(_save_runnables(runnable, path, loader_fn, persist_dir))
-    elif isinstance(runnable, lc_runnable_branch_type()):
```

Comment:
What is the intension for this deletion? Can RunnableBranch be nested?

Reply:
Oh because lc_runnable_branch_type is part of lc_runnables_types 🤣

Reply:
Aha! Sorry for missing that🙏